TesseractOCR/G8Tesseract.mm

```diff
@@ -51,6 +51,22 @@ @implementation G8Tesseract
 
 @synthesize absoluteDataPath=_absoluteDataPath;
```

Comment:
I think one uncovered line is not so bad. Is it so principal to cover this line? If so I think better way is to use dot-notation in initializer.